tests/nodenext/index.ts

```diff
@@ -0,0 +1,93 @@
+// This test does not use Jest because we want to test
+// that the output of the build process works under native Node.js ESM syntax
+// At this moment Jest does not support ESM syntax without transpiling
```

Comment:
Making the date and Jest version explicit in this comment may help future contributors.

Reply:
@avaly I copied these comments from ESM test. Should we change both?

Reply:
done!